### PR TITLE
add marketplace banner that links to strapi awesome

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

Adds a banner on the marketplace page that links to strapi awesome: https://github.com/strapi/awesome-strapi 

### Why is it needed?

Helps people find community plugins

### How to test it?

Run strapi admin locally and go to the marketplace page

### CODE REVIEW

**Review by derrickmehaffy — COMMENTED** [review]
LGTM, thanks for adding this!

**Review by mfrachet — DISMISSED** [review]
LGTM 👍🏻

File: packages/strapi-admin/admin/src/containers/MarketplacePage/PluginBanner/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Wrapper from './Wrapper';
+import LogoStrapi from '../../../assets/images/banner_strapi-rocket.png';
+
+const PluginBanner = () => {
+  return (
+    <Wrapper>
+      <img className="bannerImage" src={LogoStrapi} alt="A strapi rocket logo" />
```
[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-admin/admin/src/containers/MarketplacePage/PluginBanner/index.js:9]: as you did for the `<FormattedMessage id="app.components.PluginBanner" />`

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

**Review by HichamELBSI — APPROVED** [review]
LGTM good job @markkaylor !

**Pull request merged**